### PR TITLE
Feature get arch from instance

### PR DESCRIPTION
**Pull request by dkropachev**
Get architecture from instance type and look at the image of this architecture

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/utils/aws_utils.py [hunk 1]
```diff
@@ -311,3 +311,9 @@ def ec2_ami_get_root_device_name(image_id, region):
     except (TypeError, ClientError) as exc:
         raise AssertionError(f"Image '{image_id}' details not found in '{region}'") from exc
     return None
+
+
+def get_arch_from_instance_type(instance_type: str) -> AwsArchType:
+    if instance_type.startswith('a1.'):
```
Comment (fgelcer) [inline sdcm/utils/aws_utils.py:317]: that is not the only type of ARM instances..
in my test i'm using `im4gn.2xlarge` and also i know about `is4gen.4xlarge`

Comment (fgelcer) [reply sdcm/utils/aws_utils.py:317]: there might be others as well

**Review by aleksbykov — DISMISSED** [review]

File: sdcm/utils/aws_utils.py (hunk 1 above)
Comment (dkropachev) [reply sdcm/utils/aws_utils.py]: thanks. fixed

**Review by fgelcer — APPROVED** [review]
LGTM

**Review by soyacz — APPROVED** [review]

**Pull request merged**